### DISCUSSION TITLE
Pit() Not Working

### DISCUSSION THREAD

**Discussion by dahalji** [bastianvandenbout/LISEM#16] [Q&A]
Hi, LISEM community members!
I was trying to find catchments as shown in the documents, but it just shows
Info: Line: (1 (1) : Compiling void main()
Error: Line: (4 (30) : No matching symbol 'Pit'
Compilation Error
what am i doing wrong

**Comment by bastianvandenbout** [comment]
Hi,

Apologies for the late reply, I heard from other users, the function is named Outlet(...).
There are two versions, Outlet(ldd, pits)
and Outlet(ldd).
The first one has a bug that prevents any usage.
The second one works fine.
In the upcoming 0.2.5 the function will be fixed.

Kind regards,
Bastian
